components/brave_search_conversion/pref_names.h

```diff
@@ -14,7 +14,7 @@ inline constexpr char kDismissed[] = "brave.brave_search_conversion.dismissed";
 inline constexpr char kMaybeLaterClickedTime[] =
     "brave.brave_search_conversion.maybe_later_clicked_time";
 
-// Index ranges from 0 to 3 and it's matched DDGTypeA to DDGTypeD.
+// Index ranges from 0 to 1 and it's matched DDGTypeC to DDGTypeD.
 // It's the type index for now.
 inline constexpr char kDDGBannerTypeIndex[] =
     "brave.brave_search_conversion.ddg_banner_type_index";
```

Comment:
Should we migrate inddex 2, 3 for previous users?

Reply:
Don't need to. This dog promotion is not released yet.